src/index.html

```diff
@@ -1,13 +1,509 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600;700&display=swap" rel="stylesheet">
     <title>Dia</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Dia</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="conteiner">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class=" top-bar__logo-link">
+              <img
+                src="./images/logo/Air-logo.png"
+                alt="Air"
+                class="top-bar__logo"
+              >
+            </a>
+
+            <div class="top-bar__icons">
+              <a href="#menu" class="icon icon--menu"></a>
+
+              <nav class="nav menu__nav">
+                <ul class="nav__list">
+                  <li class="nav__item">
+                    <a class="nav__link" href="#Who-we-are">About us</a>
+                  </li>
+
+                  <li class="nav__item">
+                    <a class="nav__link" href="#services">Services</a>
+                  </li>
+
+                  <li class="nav__item">
+                    <a class="nav__link" href="#testimonials">Testimonials</a>
+                  </li>
+
+                  <li class="nav__item">
+                    <a class="nav__link" href="#contact-us">Contact us</a>
+                  </li>
+                </ul>
+              </nav>
+
+              <a href="#contact-us" class=" icon--phone">Hire us</a>
+            </div>
+          </div>
+
+          <div class="header__bottom">
+            <h1 class="header__title">Strategic Agency</h1>
+
+            <p class="header__description">We believe in the power of bold ideas that can solve business challenges.</p>
+
+            <a class="header__button button" href="#our-expertise">Learn more</a>
+          </div>
+        </div>
+      </div>
+
+      <div class="slider ">
+        <input
+          type="radio"
+          name="button"
+          class="slider__button slider__button--left"
+        >
+
+        <input
+          type="radio"
+          name="button"
+          class="slider__button slider__button--right"
+        >
+
+        <img
+          src="./images/slider/photo-slider-1.png"
+          alt="img"
+          class="slider__img slider__img--1"
+        >
+
+        <img
+          src="./images/slider/photo-slider-2.png"
+          alt="img"
+          class="slider__img slider__img--2"
+        >
+
+        <div class="slider__intro">
+          <h3 class="slider__intro__title">Intro</h3>
+
+          <p class="slider__intro__description">By the same illusion which lifts the <br> horizon.</p>
+        </div>
+
+      </div>
+    </header>
+
+    <main class="main">
+      <div class="main__content">
+        <section class="who-we-are" id="Who-we-are">
+          <h2 class="
+          who-we-are__title
+          section-title
+          ">
```

Comment:
Need to rewrite like this everywhere
```suggestion
          <h2
            class="
              who-we-are__title
              section-title
            "
          >
```